src/modules/displayTasks.js

```diff
@@ -62,8 +62,20 @@ const displayTasks = () => {
   }
 
   /* Delete completed */
+  const deleteCompleted = document.querySelector('.completed-text');
   const allCheckBoxes = document.querySelectorAll('input[type=checkbox]');
 
+  function clearCompleted(elem) {
+    return elem.completed !== true;
+  }
+
+  deleteCompleted.addEventListener('click', () => {
+    const data = taskArray.filter(clearCompleted);
+    data.sort((a, b) => a.index - b.index);
+    localStorage.setItem('taskInput', JSON.stringify(data));
+    window.location.reload();
+  });
+
   for (let i = 0; i < allCheckBoxes.length; i += 1) {
     allCheckBoxes[i].addEventListener('click', (e) => {
       checkBoxesStatus(e);
```

Comment:
- [ ] Great job :+1: Implementing the functionality for the checkbox status but from the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md), you should use the `change` event on the checkbox instead of the `click`. I think the `click` is normally for buttons, for more info on how to do that please visit [checkbox-change-event-with-javascript](https://www.techiedelight.com/bind-to-checkbox-change-event-with-javascript/)